tools/cli/src/cli/session.rs

```diff
@@ -228,9 +238,13 @@ impl LoginOpt {
 
     pub async fn exec(&self) {
         let mut client = self.copt.to_unauth_client();
-
-        // TODO: remove this anon, nobody should do default anonymous
-        let username = self.copt.username.as_deref().unwrap_or("anonymous");
+        let username = match self.copt.username.as_deref() {
+            Some(val) => val,
+            None => {
+                error!("Please specify a username to login with!");
```

Comment:
Should say "specify a username with `-D` to login"

Reply:
Did you push the update with this in it?

Reply:
I have now!